Containerfile

```diff
@@ -1,19 +1,15 @@
-FROM registry.fedoraproject.org/fedora:39 AS builder
-RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
+FROM registry.access.redhat.com/ubi9/ubi:latest AS builder
```

Comment:
Would you mind pinning the version instead of using the latest tag?

Reply:
You want to use renovate to update it?

I dunno...part of the idea here is: this is RHEL, the change that this breaks us should be 0.0001% - if it does we have totally failed at our jobs.